app/models/user.js

```diff
@@ -30,5 +30,16 @@ module.exports = (sequelize, DataTypes) => {
       underscored: true
     }
   );
+
+  User.getAll = (props, limit = 5, offset = 0) => {
```

Comment:
I think it would be nice to move this paging default values to the config file, so you can freely access them from anywhere, just in case you need to paginate queries from other models in the future (Spoiler Alert: you will have to! :laughing: )